Nataly/src/components/error-empty-results/error-empty-results.component.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import "./error-empty-results.styles.scss"
+
+import {IErrorEmptyResultsProps} from "../../interfaces/interfaces"
```

Comment:
If you not share an interface it is better to keep it in component file, just not to navigate through files all the time

Reply:
I don't see using of props here. Remove it if you don't use it